example/themes/business/src/components/DriverMap/index.tsx

```diff
@@ -475,9 +476,19 @@ export const DriverMap = (props: GoogleMapsParams) => {
                 src={theme.images.general.arrow_distance}
                 style={styles.arrowDistance}
               />
-              <OText size={12} numberOfLines={3}>{`${(
-                distancesFromTwoPlacesKm
-              ).toFixed(2)} ${t('KM', 'KM')}`}</OText>
+              {distanceUnit === 'mi' ? (
```

Comment:
Create a util function for this, A function when you pass the `distanceUnit` and return the value to show